tests/testsuite/elixir.rs

```diff
@@ -0,0 +1,54 @@
+use ansi_term::Color;
```

Comment:
You'll need to get the Dockerfile and github actions sorted out before these will pass in CI.